avocado/utils/asset.py

```diff
@@ -98,6 +98,7 @@ def _create_hash_file(self, asset_path):
         :param asset_path: full path of the asset file.
         """
         result = crypto.hash_file(asset_path, algorithm=self.algorithm)
+        self.asset_hash = result
```

Comment:
Hi @wangxiaorou92 thanks for contributing here. Could you please better describe the actual problem that you are solving here? Do you have an issue to link with details?